### PR TITLE
HRSPLT-437 system Performance Management list-of-links JSON

### PR DESCRIPTION
**Pull request by apetro**
Add a new Portlet with a new JSON resource URL vending JSON suitable for driving a `list-of-links` widget requested in [HRSPLT-437](https://jira.doit.wisc.edu/jira/browse/HRSPLT-437).

### CODE REVIEW

**Comment by apetro** [comment]
Pretty sure I just don't agree with Code Climate that the "issues" it's flagging are issues.

File: hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/listoflinks/Link.java [hunk 1]
```diff
@@ -10,6 +10,12 @@
   private String target;
   private String icon;
 
+  public Link(String title, String href, String target, String icon) {
```
Comment (apetro) [inline hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/listoflinks/Link.java:13]: This removed the implied no-arg constructor, restored immediately following this merge in 8fe93d8e8fe84ea2167578051bffa5bbaced0fce .

File: hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/performance/PerformanceLinksController.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        logger.warn(
+          "HRS URL [" + HrsUrlDao.SELF_PERFORMANCE_KEY + "] expected" +
+          " but not found and so could not be offered to " + emplId);
+      }
+    }
+
+    if (roles.contains(ROLE_LINK_MANAGE_PERFORMANCE)) {
+      if (StringUtils.isNotBlank(othersPerformanceUrl)) {
+        linkList.add(
+            new Link("Manage", othersPerformanceUrl, "_blank", "assignment_group"));
```
Comment (jimhelwig) [inline hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/performance/PerformanceLinksController.java:93]: Should be "Manager"

Comment (apetro) [reply hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/performance/PerformanceLinksController.java:93]: Fixed in #192

**Pull request merged**